src/components/ProjectList/ProjectTable/ProjectTable.tsx

```diff
@@ -1,35 +1,27 @@
 import { messages } from '@camp/messages';
 import { createTestAttr } from '@camp/utils';
 import { Table, useMantineTheme } from '@mantine/core';
+import { useProjectListQuery } from '../../../data-layer/operations/__generated__/typesAndHooks';
```

Comment:
you should export `useProjectListQuery` in data-layer and please import it from `'@camp/data-layer'`